internal/co_coders/criteria/collab_style_test.go

```diff
@@ -5,37 +5,43 @@ import (
 	"testing"
 )
 
+var (
+	Team = NewCollabStyle("Team")
```

Comment:
Outside the context of an "enum" I'm not sure it makes sense to have these as vars like this. 

I would prefer to just make a list like 
`collabStyles = newCollabStyles("Team","Pair","Mob")`
Is there something I'm missing?

Reply:
I agree, that would be better.